src/user/user.entity.ts

```diff
@@ -1,177 +1,100 @@
-import {BaseEntity,Column,Entity,Index,JoinColumn,JoinTable,ManyToMany,ManyToOne,OneToMany,OneToOne,PrimaryColumn,PrimaryGeneratedColumn,RelationId} from "typeorm";
-import {Flight} from "../flight/flight.entity";
-import {Glider} from "../glider/glider.entity";
-import {Place} from "../place/place.entity";
+import { Column, Entity, Index, OneToMany, PrimaryGeneratedColumn } from "typeorm";
+import { Flight } from "../flight/flight.entity";
+import { Glider } from "../glider/glider.entity";
+import { Place } from "../place/place.entity";
+
+@Index("idx_16606_idx_e12875dfb3b1d92d7d7c5377e2", ["email"], { unique: true })
+@Index("idx_16606_email", ["email"], { unique: true })
+@Index("idx_16606_idx_846d5513f58dc765ecc299582a", ["emailCanonical"], {
+  unique: true,
+})
+@Index("idx_16606_uniq_8d93d649a0d96fbf", ["emailCanonical"], { unique: true })
+@Index("idx_16606_primary", ["id"], { unique: true })
+@Index("idx_16606_uniq_8d93d64992fc23a8", ["usernameCanonical"], {
+  unique: true,
+})
+@Index("idx_16606_idx_ed85f2a29d842b9b0dfefaba34", ["usernameCanonical"], {
+  unique: true,
+})
+@Entity("user")
```

Comment:
Does this work without problems? I remember having problems with 'User' Tables in Postgres. But might just be something specific to Spring / JPA :)

Reply:
I also had a problem with this table. User is a reserved table in Postgres. It is however possible to use it but you have to specify the schema used